File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 1                        # nums[0] is always unique
        for i in range(1, len(nums)):
            if nums[i] != nums[k - 1]:   # nums[k-1] is the last value we kept
                nums[k] = nums[i]
                k += 1
        return k

File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
      int n = nums.length;

      for (int i =0; i < n; i++) {
        for (int j = i + 1; j < nums.length; j++){
            if (nums[i] == nums[j]){
                return true;
            }
        }
        
      } 
      return false;
    }
}